docs/tutorials/helm.md

```diff
@@ -0,0 +1,226 @@
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl), the Kubernetes CLI.
+- [`helm`](https://helm.sh/), the package manager for Kubernetes.
+
+## Create a local kubernetes cluster
+
+> 🤚 If you already have a fully fonctional Kubernetes cluster to work with,
+> you can use it and proceed to the next section after have created the
+> `development-ralph` namespace.
+
+Thanks to K3D, creating a local, docker-based Kubernetes cluster can be
+achieved using the following command:
+
+```sh
+k3d cluster create ralph
+```
+
+This cluster is called `ralph`, you can check that it has been created using:
+
+```
+k3d cluster list
+```
+
+This command should print the following output (note that the cluster is up and
+running):
+
+```
+NAME     SERVERS   AGENTS   LOADBALANCER
+ralph    1/1       0/0      true
+```
+
+We will now create a new namespace to work on:
+
+```
+kubectl create namespace development-ralph
+```
+
+And we will set this namespace as the default namespace for future `kubectl`
+commands:
+
+```
+kubectl config set-context --current --namespace=development-ralph
+```
+
+Our cluster is now ready to host our learning analytics stack!
+
+## Deploy a learning analytics stack
+
+The proposed learning analytics stack is composed of three different
+applications:
+
+1. [Ralph](https://github.com/openfun/ralph), a lightweight LRS,
+2. [Elasticsearch](https://www.elastic.co/elasticsearch/), our data lake that will store learning traces (xAPI),
+3. [Grafana](https://grafana.com/grafana/), a dashboarding solution.
+
+### Elasticsearch
+
+As Elasticsearch is required by Ralph and Grafana, we will deploy an
+Elasticsearch cluster using the official elasticsearch Helm Chart:
+
+```sh
+# Add the offical Elasticsearch Helm repository
+helm repo add elastic https://helm.elastic.co
+
+# Update the list of available charts
+helm repo update
+
+# Deploy a "simple" Elasticsearch cluster with 3 nodes
+helm install \
+  es-quickstart \
+  elastic/elasticsearch \
+  -n development-ralph \
+  --set antiAffinity=soft
+```
+
+> By default, this Helm chart will deploy three (3) Elasticsearch nodes in
+> three (3) differents Kubernetes nodes. Since we only have a single-node
+> Kubernetes cluster, we set the `antiAffinity` value to `soft` to allow our
+> deployment scenario. Feel free to adapt this option value depending on your
+> cluster constraints.
+
+Once installed, you can check pods status using the following command (and wait
+for them to get ready):
+
+```
+kubectl get pods -l app=elasticsearch-master -w
+```
+
+Once the Elasticsearch cluster is up and running, we will need to get
+automatically generated credentials to connect other applications to the
+service.
+
+For this tutorial, we will fetch `elastic` user's password using the following
+command:
+
+```
+kubectl get secrets \
+  elasticsearch-master-credentials \
+  -ojsonpath='{.data.password}' | \
+    base64 -d
```

Comment:
You could pipe this command into the one line 145. IMO, it's easier when trying the tutorial for the first time

Reply:
As we need the password on multiple occasions, I think it's better to keep it this way!